festim/hydrogen_transport_problem.py

```diff
@@ -787,23 +787,26 @@ def post_processing(self):
         for export in self.exports:
             # TODO if export type derived quantity
             if isinstance(export, F.SurfaceQuantity):
-                if isinstance(export, F.SurfaceFlux):
+                if (
+                    isinstance(export, F.SurfaceFlux)
+                    or isinstance(export, F.TotalSurface)
+                    or isinstance(export, F.AverageSurface)
+                ):
```

Comment:
You can simplify this to:

```suggestion
                if isinstance(export, (F.SurfaceFlux, F.TotalSurface, F.AverageSurface)):
```